yaas/cmd.py

```diff
@@ -12,18 +20,22 @@ def version(parser, args):
     """print the yaas version"""
     print YAAS_VERSION
 
-def hello(parser, args):
-    """show an elephant"""
-    print "                .-.._       "
-    print "          __  /`     '.     "
-    print "       .-'  `/   (   a \    "
-    print "      /      (    \,_   \   "
-    print "     /|       '---` |\ =|   "
-    print "    ` \    /__.-/  /  | |   "
-    print "       |  / / \ \  \   \_\  "
-    print "       |__|_|  |_|__\       "
+def _response_hook(res, *args, **kwargs):
```

Comment:
Doesn't need to be underscore-prefixed (that's only the convention when naming fields in objects that contain delicate local state). Also should probably live in a `util.py`.

In general, functions should be named with verbal phrases (`print_response` not `response_hook`).